apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -324,17 +324,15 @@ def prepare_tag(
                 raise ValueError(
                     f'Tag "{group_tag}" used in multiple places. The tag attribute for each <pl-answer> and <pl-block-group> must be unique.'
                 )
-            else:
-                used_tags.add(group_tag)
+            used_tags.add(group_tag)
 
             for grouped_tag in html_tags:
                 if html_tags.tag is Comment:
                     continue
-                else:
-                    prepare_tag(
-                        grouped_tag, index, {"tag": group_tag, "depends": group_depends}
-                    )
-                    index += 1
+                prepare_tag(
```

Comment:
This is where I see more value in the `if...continue` linter -- I like this change.